app/src/main/admin/stores/createAdminStore.ts

```diff
@@ -0,0 +1,13 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+
+import reducers from '../reducers/reducers';
+
+export function createAdminStore() {
+    return createStore(
+        reducers,
+        applyMiddleware(thunk),
```

Comment:
Can you remind me what this does?

Reply:
Even better, could you add a comment that explains it! :)